### ISSUE TITLE
Should email attachments contained in body_multipart be a source of a relationship?

### ISSUE THREAD

**Issue by dwjohnston** [oasis-open/cti-stix-visualization#49]
For posterity, my code is slightly behind, but I believe this is still an issue with current master. I'm working from [this commit](https://github.com/oasis-open/cti-stix-visualization/tree/96a2bafcb8f9bb548a9fe3fba6c2cdc19b882dad). 

Example bundle: 

```
[


  {
    id: 'email-addr--ea515a5f-8051-5558-a48c-1828f5dcbf8a',
    spec_version: '2.1',
    type: 'email-addr',
    value: 'example@example.com'
  },
  {
    hashes: {
      MD5: 'e04eec8e96075042cc1b5a9fb6c0df6e',
      'SHA-1': '6021bd62a5d8658e33bc7c26af7f4979963da43e',
      'SHA-256':
        'abd1b33de3e39ae651bbd4bdc6300ba4005029b369c6c9eb67e8bb1392faa8b3'
    },
    id: 'artifact--55fe7d9e-225c-5d4c-a108-8ea587d14b3c',   // <-- The attachment
    mime_type: 'application/pdf',
    spec_version: '2.1',
    type: 'artifact',
    payload_bin:
      'BASE64SHORTPAYLOAD1111111111111111111112111111==',
  },

  {
    body_multipart: [

      {
        body_raw_ref: 'artifact--55fe7d9e-225c-5d4c-a108-8ea587d14b3c', // Should link here
        content_disposition: 'attachment',
        content_type: 'application/pdf'
      },

    ],
    date: '2018-08-21T09:14:39-07:00',
    from_ref: 'email-addr--ea515a5f-8051-5558-a48c-1828f5dcbf8a',
    id: 'email-message--184d0768-b86b-5c94-97eb-00af08aca2de',
    is_multipart: true,
    spec_version: '2.1',
    subject: '0day Research',
    to_refs: ['email-addr--c8c6d744-13ad-59d8-adbd-596c8bed4f6d'],
    type: 'email-message'
  },

  {
    id: 'email-addr--c8c6d744-13ad-59d8-adbd-596c8bed4f6d',
    spec_version: '2.1',
    type: 'email-addr',
    value: 'foo@bar.com'
  }
]
```

That is, I have an artifact (an email attachment) that I want show the source of (the email). 

What I currently get is an orphaned artifact node in the visualisation: 

<img width="525" alt="Screen Shot 2022-03-28 at 7 07 37 pm" src="https://user-images.githubusercontent.com/2467377/160354670-a933b29c-e9f1-4841-a16d-1e8de664489c.png">

Whereas I want the relationship to show as an edge: 

<img width="525" alt="Screen Shot 2022-03-28 at 7 07 54 pm" src="https://user-images.githubusercontent.com/2467377/160354738-f8f1d003-a9b4-4cbe-9cf9-899edf4b5098.png">

I've already got this working, so happy to create a PR, just want to know if this is desirable behaviour. 

(As an aside, it would also be nice to make the distance for for attachment relationships shorter than regular relationships).

**Comment by clenk** [comment]
Yes, that would be desirable behavior - I would love to see a PR!

I also like the idea of making embedded relationships (*_ref/s) use shorter arrows. However, we're currently investigating some other changes to how graphs are displayed. Once those changes are implemented we can revisit this idea. Otherwise we might have to redo it.

**Comment by ejratl** [comment]
For the purposes of testing the associated PR, I adapted the example above as in the attached email-test-bundle-49.json (txt for GitHub). It tests out correctly.
[email-test-bundle-49.json.txt](https://github.com/oasis-open/cti-stix-visualization/files/9246025/email-test-bundle-49.json.txt)

**Comment by ejratl** [comment]
This issue was fixed via PR #52
